spec/schema.json

```diff
@@ -53,6 +53,12 @@
             "schema": {
               "$ref": "#/definitions/ResourceCreated"
             }
+          },
+          "406": {
```

Comment:
According to Mozilla: 406 Not Acceptable client error response code indicates that the server cannot produce a response matching the list of acceptable values defined in the request's proactive content negotiation headers, and that the server is unwilling to supply a default representation.

As this error doesn't relate to content types i'd be hesitant about using this code. I think a simple 400, with a body containing an ineligible code would be more than descriptive enough

Reply:
There don't seem to be any other 4xx errors that are suitable so will opt for 400.